### PR TITLE
Azure Monitor - Updated example code

### PR DESCRIPTION
**Pull request by jawache**
#sign-off as per @TomArcher

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/monitor.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-7/docs-ref-services/monitor.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C17%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707171622025726-96%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@jawache  @johnpapa Merged

File: docs-ref-services/monitor.md [hunk 1]
```diff
@@ -13,22 +13,43 @@ ms.devlang: nodejs
...
+# Azure Monitor Packages for Node.js
 
 ## Overview
 Cloud applications are complex with many moving parts. Monitoring provides data to ensure that your application stays up and running in a healthy state. It also helps you to stave off potential problems or troubleshoot past ones. In addition, you can use monitoring data to gain deep insights about your application. That knowledge can help you to improve application performance or maintainability, or automate actions that would otherwise require manual intervention.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure Monitor modules for Node.js
+### Install npm module
```
Comment (johnpapa) [inline docs-ref-services/monitor.md]: module --> package

add a sentence here ...

> Install the Azure monitor module for Node.js

File: docs-ref-services/monitor.md [hunk 2]
```diff
@@ -13,22 +13,43 @@ ms.devlang: nodejs
...
-### Management
 ```bash
 npm install azure-arm-insights
 ```
 
-##Example
+### Example
 
-##Samples
+This snippet prints all the alerting rules associated with a resource group.
```
Comment (johnpapa) [inline docs-ref-services/monitor.md]: snippet --> code example

**Comment by johnpapa** [comment]
ugh .... we need to not sign-off on these til we do reviews.

@jawache please make a new PR and make these fixes.

in the future let's not mark sign-off til its been reviewed

**Pull request merged**